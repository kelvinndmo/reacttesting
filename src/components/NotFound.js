import React from "react";
import { Link } from "react-router-dom";

const NotFound = () => (
  <div>
    <h3>
      404! <Link to="/">- The page ure looking for is unavaibale</Link>
    </h3>
  </div>
);

export default NotFound;
