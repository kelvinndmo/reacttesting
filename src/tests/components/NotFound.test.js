import React from "react";
import NotFound from "../../components/NotFound";
import { shallow } from "enzyme";

test("should render not found if page not found", () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
