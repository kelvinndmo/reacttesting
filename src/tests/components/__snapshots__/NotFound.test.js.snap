// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found if page not found 1`] = `
<div>
  <h3>
    404! 
    <Link
      to="/"
    >
      - The page ure looking for is unavaibale
    </Link>
  </h3>
</div>
`;
